main.py

```diff
@@ -335,40 +354,46 @@ def OWMCITY(self, _: Event | None = None) -> None:
             [
                 f"Weather: {weather.status} ~ {weather.detailed_status}",
                 f"Current Temperature: {temperature.get('temp', None):.2f}°C",
-                f"Maximum Temperature: {temperature.get('temp_max', None):.2f}°C",
-                f"Minimum Temperature: {temperature.get('temp_min', None):.2f}°C",
+                f"Maximum Temperature: {
+                    temperature.get('temp_max', None):.2f}°C",
```

Comment:
Put the rest of the actual string on its own line. Separate variables and string if you want more than one line